### DISCUSSION TITLE
Questions about arachnophobia friendly mode

### DISCUSSION THREAD

**Discussion by ShyASz** [Aelto/W3_RandomEncounters_Tweaks#122] [Q&A] [answered]
I want to figure out whether it will replace spiders with nekker only for spiders generated in random encounter mods or for spiders for the entire game. To my embarrassment, my fear of insectoids prevent me from playing well. If I encounter insectoids such as spiders or Endrega workers while doing a mission, my instincts will make me choose to escape rather than fight, in fact I met once today and did. There's no doubt that Geralt is strong and brave, but forgive me for not being able to manipulate him normally to kill these guys. So I raised that question and look forward to getting an answer.
I have to admit that The Witcher 3 was very attractive to me and I really wanted to keep playing. To achieve this, I've also been looking for a solution to the problem (unfortunately I can't start with my own fears). Before I officially played, I learned about the existence of spiders and insectoids in the game, and found several mods to replace spiders in nexusmods. I used to think spiders had to be avoided, and I might be able to overcome other insectoids, but today's events have hit me in the face. Maybe I could avoid these creatures by searching for their haunting points, but unfortunately I tried them but haven't found them so far. If, in the end, I can't find an effective way to avoid these creatures, I think my journey will end there

**Comment by Aelto** [comment, marked as answer]
Hello, the arachnophobia mode for RER only works for the monsters spawned by the mod itself. It is an internal check, which replaces the spider entries with nekkers when the mod is loading the data. So you can still see spiders from the vanilla game, RER won't do anything to them.

I took a look at the mods you talked about on nexusmods, from what i understand they should also work for the spiders spawned by RER.

**Reply by ShyASz** [reply]
Thank you for your reply and for paying the highest respect to you selfless mod workers
